third_party/tutorial.proto

```diff
@@ -27,16 +27,9 @@ message Tutorial {
 message Step {
   string title = 4;
   map<string, string> metadata = 2;
-  repeated Content content = 3;
+  repeated BlockContent content = 5;
 
-  reserved 1;
-};
-
-message Content {
-  oneof content {
-    InlineContent inline_content = 1;
-    BlockContent block_content = 2;
-  }
+  reserved 1, 3;
```

Comment:
3?

Reply:
3 was `content` with type `Content`, which is deleted on this PR. reusing name.